docs/pages/eas/environment-variables.mdx

```diff
@@ -0,0 +1,193 @@
+These environment variables are available to your build jobs, workflow jobs and updates for this project. They can also be pulled locally for development if their visibility setting allows it.
+
+## Account-wide environment variables
+
+Account-wide environment variables are environment variables that are available across all of your projects in your EAS account.
+You can view and manage them by navigating to [Environment Variables](https://expo.dev/accounts/[account]/settings/environment-variables) section of the account settings menu.
+They are available to your build jobs, workflow jobs and updates, together with project-wide variables for this project.
+You can pull them locally or read outside of EAS servers if their visibility setting allows it.
+
+## Environments
+
+Currently EAS supports three environments for environment variables: `development`, `preview` and `production`.
+Environments are independent sets of environment variables that can be used to customize your app in different contexts.
+For example, you might want to use different API keys for development and production, or different bundle identifiers for different App Store releases.
+Environments allows you to do so. Every EAS Build and EAS Workflow job runs using environment variables from one of the available environments.
+Environments can also be used for updates, allowing you to easily use the same set of environment variables as for your build jobs, when creating an update using `eas update` command with `--environment` flag.
+
+The environment variables defined on EAS servers can be assigned to mutiple environments and have the same value across all of them, or be created for a single environment, so they can have different values for each environment.
+
+## Visibility
+
+We distinguish between three visibility settings for environment variables:
+
+| Visibility | Description                                                                                                                   |
+| ---------- | ----------------------------------------------------------------------------------------------------------------------------- |
+| Plain text | Visible on the website, in EAS CLI and in logs.                                                                               |
+| Sensitive  | Obfuscated in build and workflow jobs logs. You can use the toggle to make them visible on the website. Readable in EAS CLI.  |
+| Secret     | Not readable outside of the EAS servers, including on the website and in EAS CLI. Obfuscated in build and workflow jobs logs. |
+
+> **warning** Always remember that **anything that is included in your client side code should be considered public and readable to any individual that can run the application**.
+> Secret type environment variables are intended to be used to provide values to an EAS Build or EAS Workflow job so that they may be used during the execution.
```

Comment:
```suggestion
> Secret type environment variables are intended to be used to provide values to an EAS Build or Workflows job so that they may be used to alter how a job runs.
```